staging/src/k8s.io/apiserver/pkg/registry/apiserver/externaladmissionhookconfiguration/doc.go

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package externaladmissionhookconfiguration // import "k8s.io/kubernetes/pkg/registry/apiserver/externaladmissionhookconfiguration"
```

Comment:
> Wow I really object to having a specific API type living in the generic apiserver's registry package. @deads2k did you really want this? ..

I really want to have separate buckets for "generic API server types" and "kube-apiserver types".  The types are two distinct buckets.  It's easy to have separate buckets and later move them around.  It's really hard to put it all into one bucket and then later try to separate some stuff that was mixed in the bucket.  Because of that, putting them here is much easier to change than putting them in the main repo.

The first thing you're going to do is claim that the apiserver needs them and so you'll group them in a `k8s.io/apis` bucket.